docs/docs/tutorials/contract_tutorials/advanced/token_bridge/4_typescript_glue_code.md

```diff
@@ -35,27 +38,31 @@ We will write two tests:
 Open `cross_chain_messaging.test.ts` and paste the initial description of the test:
 
 ```typescript
-import { expect, jest} from '@jest/globals'
-import { AccountWallet, AztecAddress, DebugLogger, EthAddress, Fr, computeAuthWitMessageHash, createDebugLogger, createPXEClient, waitForPXE } from '@aztec/aztec.js';
+import { describe, beforeEach, expect, jest, it} from '@jest/globals'
+import { AccountWallet, AztecAddress, type DebugLogger, EthAddress, Fr, computeAuthWitMessageHash, createDebugLogger, createPXEClient, waitForPXE, L1ToL2Message, L1Actor, L2Actor } from '@aztec/aztec.js';
 import { getInitialTestAccountsWallets } from '@aztec/accounts/testing';
 import { TokenContract } from '@aztec/noir-contracts.js/Token';
+import { sha256ToField } from '@aztec/foundation/crypto';
 import { TokenBridgeContract } from './fixtures/TokenBridge.js';
 import { createAztecNodeClient } from '@aztec/circuit-types';
 
-import { CrossChainTestHarness } from './shared/cross_chain_test_harness.js';
+import { CrossChainTestHarness } from '../shared/cross_chain_test_harness.js';
```

Comment:
```suggestion
import { CrossChainTestHarness } from './shared/cross_chain_test_harness.js';
```

I think the flow on the docs makes this be at the same level as shared, so one dot only